README.md

```diff
@@ -83,6 +83,7 @@ use {
       default_register_macros = 'q',
       enable_macro_history = true,
       content_spec_column = false,
+      display_macro_raw_bytes_as_keycodes = false,
```

Comment:
If we make this default to true maybe we can invert the setting and call it `disable_keycodes_parsing`?

Reply:
Cool. I like that name